src/app/app-routing.module.ts

```diff
@@ -23,7 +25,12 @@ const routes: Routes = [
     canActivate: [UsersGuard],
     loadChildren: './modules/upload-status/upload-csv.module#UploadCSVModule'
   },
-  { path: 'home', component: HomeComponent, data: { title: [{name: 'Admin Console', link: '/home'}]} },
+  {
+    path: 'entities',
+    canActivate: [UsersGuard],
```

Comment:
this should be enabled only for platform admins

Reply:
ok , changed